.github/CODEOWNERS

```diff
@@ -7,29 +7,31 @@
 /tools/ @tjprescott
 /scripts/ @tjprescott
 /src/azure-cli-testsdk/ @tjprescott
+/src/azure-cli-core/ @tjprescott @yugangw-msft
 /src/command_modules/azure-cli-acr/ @djyou
 /src/command_modules/azure-cli-acs/ @rjtsdl
 /src/command_modules/azure-cli-advisor/ @Prasanna-Padmanabhan
 /src/command_modules/azure-cli-appservice/ @yugangw-msft
 /src/command_modules/azure-cli-backup/ @dragonfly91
 /src/command_modules/azure-cli-batch/ @bgklein
 /src/command_modules/azure-cli-batchai/ @AlexanderYukhanov
-/src/command_modules/azure-cli-cdn/ @tjprescott
```

Comment:
Why do we remove cdn reviewer?

Reply:
CDN is owned by this team. We don't know who on the service team owns it (but it's definitely not me!)

Reply:
@limingu I removed it per @tjprescott's suggestion, as no one on the team owns it. Sorry for the late response.